xml/System.Security.Cryptography.Pkcs/CmsSigner.xml

```diff
@@ -65,9 +65,9 @@
   
 |Property|Default value|  
 |--------------|-------------------|  
-|<xref:System.Security.Cryptography.Pkcs.CmsSigner.SignerIdentifierType%2A>|<xref:System.Security.Cryptography.Pkcs.SubjectIdentifierType?displayProperty=nameWithType>|  
-|<xref:System.Security.Cryptography.Pkcs.CmsSigner.DigestAlgorithm%2A>|1.3.14.3.2.26 (szOID_OIWSEC_sha1)|  
-|<xref:System.Security.Cryptography.Pkcs.CmsSigner.IncludeOption%2A>|<xref:System.Security.Cryptography.X509Certificates.X509IncludeOption.None> if <xref:System.Security.Cryptography.Pkcs.CmsSigner.SignerIdentifierType%2A> is <xref:System.Security.Cryptography.Pkcs.SubjectIdentifierType>,<br /><br /> <xref:System.Security.Cryptography.X509Certificates.X509IncludeOption.ExcludeRoot> otherwise|  
+|<xref:System.Security.Cryptography.Pkcs.CmsSigner.SignerIdentifierType%2A>|<xref:System.Security.Cryptography.Pkcs.SubjectIdentifierType.IssuerAndSerialNumber?displayProperty=nameWithType>|  
+|<xref:System.Security.Cryptography.Pkcs.CmsSigner.DigestAlgorithm%2A>|2.16.840.1.101.3.4.2.1 (SHA-256)|  
+|<xref:System.Security.Cryptography.Pkcs.CmsSigner.IncludeOption%2A>|<xref:System.Security.Cryptography.X509Certificates.X509IncludeOption.None?displayProperty=nameWithType> if <xref:System.Security.Cryptography.Pkcs.CmsSigner.SignerIdentifierType%2A> is <xref:System.Security.Cryptography.Pkcs.SubjectIdentifierType.NoSignature?displayProperty=nameWithType>,<br /><br /> <xref:System.Security.Cryptography.X509Certificates.X509IncludeOption.ExcludeRoot?displayProperty=nameWithType> otherwise|  
```

Comment:
This doesn't seem to make much sense since the default value for SignerIdentifierType is IssuerAndSerialNumber when using this overload. maybe this and then for each overload add the appropriate value?

```suggestion
|<xref:System.Security.Cryptography.Pkcs.CmsSigner.IncludeOption%2A>|<xref:System.Security.Cryptography.X509Certificates.X509IncludeOption.ExcludeRoot?displayProperty=nameWithType>|  
```

Reply:
I agree.  I started doing it, then decided that I'd stick with the style and undid my edits.  Should've had the courage of my convictions.

Reply:
You have good doc instincts - you should definitely trust them!